gguf-py/gguf/constants.py

```diff
@@ -25,6 +25,20 @@ class General:
         ALIGNMENT                  = "general.alignment"
         FILE_TYPE                  = "general.file_type"
 
+        # Recommended Sampler Parameters
+        SAMPLER_SEQUENCE           = "general.sampler.sequence"
+        SAMPLER_TOP_K              = "general.sampler.top_k"
+        SAMPLER_TOP_P              = "general.sampler.top_p"
+        SAMPLER_MIN_P              = "general.sampler.min_p"
+        SAMPLER_XTC_PROBABILITY    = "general.sampler.xtc_probability"
+        SAMPLER_XTC_THRESHOLD      = "general.sampler.xtc_threshold"
+        SAMPLER_TEMP               = "general.sampler.temp"
+        SAMPLER_PENALTY_LAST_N     = "general.sampler.penalty_last_n"
+        SAMPLER_PENALTY_REPEAT     = "general.sampler.penalty_repeat"
+        SAMPLER_MIROSTAT           = "general.sampler.mirostat"
+        SAMPLER_MIROSTAT_TAU       = "general.sampler.mirostat_tau"
+        SAMPLER_MIROSTAT_ETA       = "general.sampler.mirostat_eta"
+
```

Comment:
Isn't `general.sampling` a more appropriate prefix than `general.sampler`?

Usually, we use "sampler" to refer to the object that performs the sampling. Here we are referring to the sampling parameters.